packages/http/server/response_builder.mare

```diff
@@ -0,0 +1,86 @@
+:import "bytes" (ByteStreamWritable)
+
+:class ResponseBuilder
+  :let _res ByteStreamWritable
+
+  :new (target ByteStreamWritable)
+    @_res = target
+    @_res << b"HTTP/1.1 " // TODO: I don't like inserting the version here, but carry on...
+
+  // HTTP/1.1 response status list from RFC2616
+  // https://datatracker.ietf.org/doc/html/rfc2616#section-6.1.1
+  //
+  // 1xx Informational
+  :fun ref status_continue:            @_res << b"100 Continue\r\n"
```

Comment:
If you add `@` to the end of all these methods, it will make them easily chainable at the call site.
On these one-line methods, you can use a comma to separate the `@` expression if you want to keep it as a one-liner, like this:

```suggestion
  :fun ref status_continue:            @_res << b"100 Continue\r\n", @
```

Reply:
Love it. The thought cross my mind, I'll do it.